metastor/managedObjectLoader/src/main/java/org/finra/herd/metastore/managed/jobProcessor/BackLoadObjectProcessor.java

```
@@ -264,4 +292,33 @@ private void filterAlreadyAddedPartitions( JobSubmitterInfo jsi, TreeSet<String>
 		partitions.removeAll( alreadyAddedPartitions );
 	}
 
+	void runProcess( JobDefinition jd ) throws RuntimeException {
+		ProcessBuilder pb = createProcessBuilder( jd );
+		if ( Objects.nonNull( pb ) ) {
+			ExecutorService pool = Executors.newSingleThreadExecutor();
+			try {
+				log.info( "Start Task " + pb.command() );
+				Process process = pb.start();
+				Callable task = () -> new BufferedReader( new InputStreamReader( process.getInputStream() ) )
+						.lines()
+						.collect( Collectors.toList() );
+
+				Future<List<String>> future = pool.submit( task );
+				if ( !future.isDone() ) {
+					log.info( "Waiting for task to be done" );
+					future.get( 3600, TimeUnit.SECONDS );
+				}
+
+				log.info( "Task completed" + pb.command() );
+
+			} catch ( Exception e ) {
+				log.info( "Exception in smHql runProcess" + e.getMessage() );
+				throw new RuntimeException( "Update schema failed" );
```

Send a notification here...this is a silent failure